common/models/data-source-definition.js

```diff
@@ -293,11 +293,16 @@ function ready(DataSourceDefinition) {
     http: { verb: 'POST' },
   });
 
+  // XXX(sam) This could really use a comment... it looks like the purpose here
+  // is not co-processing, but to workaround the fact that so much of what
+  // loopback does depends on the CWD from which its loaded?
```

Comment:
Workspace is a LoopBack app itself. We need to run the inspected LoopBack app out-of-process for multiple reasons:
 - prevent interference between workspace and app models
 - prevent `require.resolve` caching when we want load a new revision of the app, after we made changes
 - prevent the workspace app from crashing if the LoopBack app crashes (e.g. because it cannot load a newly added connector that was not `npm install`-ed yet)